### DISCUSSION TITLE
Random user is granted access to private repository when forking for team assignments in Classroom

### DISCUSSION THREAD

**Discussion by shanem-feduni** [community/community#116480] [GitHub Education]
### Hi everyone,

I have set up some team-based assignments using the new forking-approach, using a private repository from within the organisation. But I find that when the assignment is accepted, in addition to doing what I expect (of making a fork for the team), it seems to pick a random github user (not associated to the org in any way) to add with Read permission to the repository from which it was forked. Unless those random github users are members of the team who develop GitHub Classroom, I do not understand why they are being added. I am manually removing them when I notice them in the audit log.

**Comment by samus-aran** [comment]
Thank you for bringing this to our attention. We've passed this along to our internal team to investigate. 👍
